man/systemd.exec.xml

```diff
@@ -1650,6 +1650,18 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>LockPersonality=</varname></term>
+
+        <listitem><para>Locks down the <citerefentry
+        project='man-pages'><refentrytitle>personality</refentrytitle><manvolnum>2</manvolnum></citerefentry> system
+        call so that the kernel execution domain may not be changed from the default or the personality selected with
+        <varname>Personality=</varname> directive. This may be useful to improve security, because odd personality
+        emulations may be poorly tested and source of vulnerabilities. If running in user mode, or in system mode, but
+        without the <constant>CAP_SYS_ADMIN</constant> capability (e.g. setting <varname>User=</varname>),
```

Comment:
> if the service is run without CAP_SYS_ADMIN or as a non-root user, NoNewPrivileges=yes is implied

I think this is not correct. If the service is run as a non-root user with CAP_SYS_ADMIN, then NoNewPrivileges= is not implied. I think @topimiettinen's original sentence is correct.